bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -441,6 +453,31 @@ spec:
                   requests:
                     cpu: 20m
                     memory: 300Mi
+              - name: kube-rbac-proxy
+                image: quay.io/openshift/origin-kube-rbac-proxy:latest
+                imagePullPolicy: Always
+                args:
+                  - --secure-listen-address=0.0.0.0:8443
+                  - --upstream=http://127.0.0.1:9182/
+                  - --tls-cert-file=/etc/tls/private/tls.crt
+                  - --tls-private-key-file=/etc/tls/private/tls.key
+                  - --logtostderr=true
+                  - --v=10
+                ports:
+                  - containerPort: 8443
+                    protocol: TCP
+                    name: metrics
+                resources:
+                  requests:
+                    cpu: 10m
+                    memory: 20Mi
+                volumeMounts:
+                - mountPath: /etc/tls/private
+                  name: wmco-tls
+              volumes:
+              - name: wmco-tls
+                secret:
+                  secretName: wmco-tls
```

Comment:
consider `windows-machine-config-operator-tls` to align with the name of the operator.